python/cuspatial/cuspatial/tests/spatial/distance/test_hausdorff_distance.py

```diff
@@ -14,7 +14,7 @@ def _test_hausdorff_from_list_of_spaces(spaces):
 def test_empty():
     actual = _test_hausdorff_from_list_of_spaces([])
 
-    expected = cudf.DataFrame([])
+    expected = cudf.DataFrame([], columns=cudf.Index([]))
```

Comment:
Likely an upstream cuDF change made the empty Dataframe uses a `RangeIndex` as the column names, instead of `cudf.Index`. Since semantically both represent empty column names, I simply change the gold result here to let tests pass.

Reply:
OK thanks. Sounds like once everything is passing we will need to update this PR to remove the CI changes and get this fix merged in?

Reply:
I think that's right, otherwise we might have to handle this at release time.